.github/workflows/build.yml

```diff
@@ -174,6 +174,9 @@ jobs:
 
       - name: Check that all modules can be imported
         run: |
+          # The following command checks that all modules can be imported.
+          # The output also includes a long list of modules together with the number of tests in each module.
+          # This can be ignored.
```

Comment:
Is the output intended? Do you use this information? Or should everyone ignore it?

Reply:
It's safe to ignore (but pytest doesn't support a way to hide it)

Reply:
Fine. I assume that you mean that you also do not use this information. We'll iterate the cosmetics in follow-up PRs.